app/views/users/_form.html.erb

```diff
@@ -1,51 +1,49 @@
 <div style="padding:10px;">
-<%= form_for @user, :url => {:controller => 'users', :action => @action}, :html => {:class => "row"} do |f| %>
+<%= form_for @user, :url => {:controller => 'users', :action => 'new'}, :html => {:class => "row"} do |f| %>
```

Comment:
I guess there were no changes in my pr for this file. Just it got uploaded somehow.
You have made some changes to the front end. Right?

Reply:
Yep, I haven't pushed yet.

Reply:
Should I use the old or new version of that line of code?

Reply:
I mean the whole file. Is it same?
I think we should use the older version.